### PR TITLE
ZIOS-10247: Create property animation with custom easing curves

### PR DESCRIPTION
**Pull request by alexaubry**
### Issues

To remove the `RBBAnimation` CocoaPods dependency, we needed a way to support custom easing function with interpolations in CoreAnimation.

### Solutions

We created an enumeration of standard (but non-system) easing curves (from [this website](https://easings.net)). Each of these curves are associated with a `CAMediaTimingFunction`, which is created using the control points of the cubic curve that represents the function.

We use these values in `EasingAnimation`, a keyframe animation subclass that supports "from" and "to" values, and that assigns the correct `CAMediaTimingFunction` depending on the selected easing curve. It can be bridged to `CABasicAnimation` in Objective-C.

### CODE REVIEW

**Comment by alexaubry** [comment]
@vytis @daehn 😮 - just fixed that. I used `mergepbx` to fix a merge conflict, these (null) probably come from there.

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -427,8 +429,8 @@
 		871BC36D1D34F96000DF0793 /* DeviceOrientationObserver.m in Sources */ = {isa = PBXBuildFile; fileRef = 871BC36C1D34F96000DF0793 /* DeviceOrientationObserver.m */; };
 		871BE95A1CD0C2E7002267E8 /* FBSnapshotTestCase.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 871BE9551CD0C2DD002267E8 /* FBSnapshotTestCase.framework */; };
 		871BE95B1CD0C3A5002267E8 /* OCMock.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 871BE9561CD0C2DD002267E8 /* OCMock.framework */; };
-		871BE95E1CD0C459002267E8 /* OCMock.framework in CopyFiles */ = {isa = PBXBuildFile; fileRef = 871BE9561CD0C2DD002267E8 /* OCMock.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
-		871BE95F1CD0C462002267E8 /* FBSnapshotTestCase.framework in CopyFiles */ = {isa = PBXBuildFile; fileRef = 871BE9551CD0C2DD002267E8 /* FBSnapshotTestCase.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		871BE95E1CD0C459002267E8 /* OCMock.framework in Copy Files */ = {isa = PBXBuildFile; fileRef = 871BE9561CD0C2DD002267E8 /* OCMock.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```
Comment (daehn) [inline Wire-iOS.xcodeproj/project.pbxproj]: It's `"Copy Files"` now and was `"CopyFiles"` before.

Comment (alexaubry) [reply Wire-iOS.xcodeproj/project.pbxproj]: Fixed

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1279,6 +1281,7 @@
 				871BE95F1CD0C462002267E8 /* FBSnapshotTestCase.framework in CopyFiles */,
 				871BE95E1CD0C459002267E8 /* OCMock.framework in CopyFiles */,
 			);
+			name = "Copy Files";
```
Comment (daehn) [inline Wire-iOS.xcodeproj/project.pbxproj]: Sry, I think this is the last one 😬

**Pull request merged**